ui/packages/catalog-ui-search/src/main/webapp/component/result-item/result-item.collection.tsx

```diff
@@ -99,12 +134,20 @@ class ResultItems extends React.Component<Props, State> {
     return showingResultsForFields.join(', ')
   }
 
+  rerunQuery(model: any) {
+    model.set('spellcheck', false)
```

Comment:
I have a couple concerns with this.

One is that it will mark the query as having been modified, making the save button appear.

The second is, what happens when they go to the next page of results?  Won't it reset to the spellchecked version?

Reply:
Just as a preface, I do agree with your concerns.
For making the save button appear, this already occurs without spellcheck, when you just re-run the query. In fact, you have to save the workspace every time a result is returned from a query (this is even when spellcheck is off).

For the second concern, this does not happen for now.

Reply:
Looks like this is happening on master anyways whenever you a query. 

![bug](https://user-images.githubusercontent.com/7646577/58280833-bc3ca680-7d5e-11e9-8260-d2d7dcf799fc.gif)